app/src/sharedTest/java/org/oppia/android/app/onboarding/OnboardingFragmentTest.kt

```diff
@@ -660,6 +760,33 @@ class OnboardingFragmentTest {
     }
   }
 
+  private fun <A : Activity> ActivityScenario<A>.getDimensionPixelSize(
+    @DimenRes dimenResId: Int
+  ): Int {
+    return runWithActivity { it.resources.getDimensionPixelSize(dimenResId) }
+  }
+
+  private inline fun <A : Activity, reified V : View> ActivityScenario<A>.findViewById(
+    @IdRes viewResId: Int
+  ): V {
+    return runWithActivity { it.findViewById(viewResId) }
+  }
+
+  private inline fun <reified V, A : Activity> ActivityScenario<A>.runWithActivity(
+    crossinline action: (A) -> V
+  ): V {
+    // Use Mockito to ensure the routine is actually executed before returning the result.
+    @Suppress("UNCHECKED_CAST") // The unsafe cast is necessary to make the routine generic.
+    val fakeMock: Consumer<V> = Mockito.mock(Consumer::class.java) as Consumer<V>
+    val valueCaptor = ArgumentCaptor.forClass(V::class.java)
+    onActivity { fakeMock.consume(action(it)) }
+    Mockito.verify(fakeMock).consume(valueCaptor.capture())
```

Comment:
Please directly import 'verify' for parity.

Reply:
Done.